grassroots-backend/src/grassroots-shared/Organization.dto.ts

```diff
@@ -0,0 +1,21 @@
+import { IsInt, IsNotEmpty, Min } from "class-validator";
+
+export class OrganizationResponseDTO {
+  id!: number;
+  name!: string;
+  parentId?: number;
+}
+
+export class CreateOrganizationRootDTO {
```

Comment:
I would call this `CreateOrganizationRootRequestDTO` since it's not mapped to an entity and is used as an API request type